lib/perl/Genome/Site/TGI/Extension/Sys.pm

```diff
@@ -128,6 +123,15 @@ sub _bsub_and_wait_for_completion__validate_args {
     return(\%special_args, %bsub_args);
 }
 
+sub _bsub_and_wait_for_completion__wait_on_jobs_with_lsf_dependency {
+    my ($class, $job_ids, $bsub_args) = @_;
+    local $ENV{LSF_NIOS_JOBSTATUS_INTERVAL} = 1;
+    $class->bsub(%$bsub_args,
```

Comment:
I don't think it's appropriate to reuse all the same bsub arguments for this job.